tests/cover/test_reflection.py

```diff
@@ -225,6 +225,10 @@ def test_source_of_lambda_is_pretty():
         lambda x: True
     ) == 'lambda x: True'  # pragma: no cover
 
+def test_bracket_whitespace_is_striped():
+    assert get_pretty_function_description(
+        lambda x: ( x + 1 )
+    ) == 'lambda x: (x + 1)'  # pragma: no cover
```

Comment:
Minor nitpick: We don't actually need these pragmas any more (we used to have coverage measurement on test files but no longer do, just on main files, but haven't gotten rid of them because I forgot they were there). No need to fix up the rest, but might as well not add them.